src/logger.rs

```diff
@@ -86,7 +94,7 @@ where
 					.extensions()
 					.get::<Activity>()
 					.expect("parent span didn't contain activity wtf"),
-				None => unsafe { &mut _os_activity_current as *mut _ },
+				None => unsafe { addr_of_mut!(_os_activity_current) },
```

Comment:
I fixed these while I was here, taking a `&mut` of a `static` is undefined, the compiler recommended these fixes.